src/lib/mappingUtils.js

```diff
@@ -26,29 +26,53 @@ function getTechniques(resource, resourceType) {
 function getBodySites(resource, resourceType) {
   return fhirpath.evaluate(resource, `${resourceType}.bodySite.coding.display`);
 }
+function getIdentifier(patient) {
+  const idType = fhirpath.evaluate(
+    patient,
+    "Patient.identifier.first().type.coding.first().display"
+  )[0];
+  const idSystem = fhirpath.evaluate(
+    patient,
+    "Patient.identifier.first().type.coding.first().system"
+  )[0];
+  const idValue = fhirpath.evaluate(
+    patient,
+    "Patient.identifier.first().value"
+  )[0];
+  if (idType) return `${idType}: ${idValue}`;
+  if (idSystem) return `${idValue} (System: ${idSystem})`;
+  return idValue;
+}
```

Comment:
DP1 - Can we make this return an array of strings, instead of just looking at the first identifier instance?

Reply:
Should we be considering the situation of multiple identifiers but different systems for each one?

Reply:
Having a hard time parsing your question, but I think the answer is yes. Each identifier object can have it's own display, system, or none. So this whole routine should basically just be mapped over array of all identifiers.

Reply:
Like, if there were multiple identifiers, the output could look like `Medical Record Number: id1, id2` or `Medical Record Number: id1, Medical Record Number: id2`, the latter looking very redundant, but would matter if the systems were different

Reply:
Not guaranteed by any spec, but I wouldn't expect multiple identifiers from the same codesystem (That's kinda the whole point of an identifier, to be unique w/r/t a system). So I'd prefer a redundant look in the minority cases where we do have two MRN's for example. 

What I've realized in your comment, however, is that there's a chance that comma-joining the strings can produce ambiguity in the else case; in that final return case, can we return a string that clarifies we weren't able to find a system or a display – something like `'${id} (No system provided)`.